Makefile

```diff
@@ -8,6 +11,28 @@ test: all
 jasmin:
 	make -C tools/jasmin-2.4
 
+build/j2me.js: $(BASIC_SRCS) $(JIT_SRCS)
+	@echo "Building J2ME"
+	node tools/tsc.js --sourcemap --target ES5 references.ts -d --out build/j2me.js
+
+build/jsc.js: jsc.ts build/j2me.js
+	@echo "Building J2ME JSC CLI"
+	node tools/tsc.js --sourcemap --target ES5 jsc.ts --out build/jsc.js
+
+j2me: build/j2me.js build/jsc.js
+
+aot: java j2me
+	js build/jsc.js -cp java/classes.jar -d -jf java/classes.jar -cff classes.txt > build/classes.jar.js
+	js build/jsc.js -cp java/classes.jar program.jar -d -jf program.jar -cff classes.txt > build/program.jar.js
+
+closure:
+	java -jar tools/closure.jar --language_in ECMASCRIPT5 -O SHUMWAY_OPTIMIZATIONS build/j2me.js > build/j2me.cc.js
```

Comment:
Yeah, we haven't decided yet if we're going to keep it in the repo. It's not currently a required build step, so it should be okay. I'm just trying to investigate the intermittent travis failures on the jit branch.